lib/_emerge/depgraph.py

```diff
@@ -7370,11 +7370,28 @@ def _wrapped_select_pkg_highest_available_imp(
                     ):
                         continue
 
+                    # We can choose not to install a live package from using binary
+                    # cache by disabling it with option --usepkg-exclude-live in the
+                    # emerge call.
+
+                    usepkg_exclude_live = False
+
+                    if "--usepkg-exclude-live" in self._frozen_config.myopts:
+                        usepkg_exclude_live = True
```

Comment:
We can define this variable before the loop, like this:
```
usepkg_exclude_live = "--usepkg-exclude-live" in self._frozen_config.myopts
```

Reply:
Yea, that makes sense.

Reply:
I pushed the changes.